client/benches/tps/utils.rs

```diff
@@ -56,11 +56,16 @@ impl Config {
     #[allow(clippy::expect_used, clippy::unwrap_in_result)]
     pub fn measure(self) -> Result<Tps> {
         // READY
-        let (_rt, network, _genesis_client) =
-            <Network>::start_test_with_runtime(self.peers, self.max_txs_per_block, None);
+        let (_rt, network, client) = <Network>::start_test_with_runtime(self.peers, None);
         let clients = network.clients();
         wait_for_genesis_committed(&clients, 0);
 
+        client.submit_blocking(
+            ParametersBuilder::new()
+                .parameter("MaxTransactionsInBlock", self.max_txs_per_block)?
+                .into_set_parameters(),
+        )?;
```

Comment:
A bit counter-intuitive that now this and only this parameter has to be set in the tests, but they're going to be deprecated anyway, so I don't mind.

Reply:
I wanted to preserve previous behavior of the test.
Because initially we set `max_trsansactions_in_block` in the configuration and this wont work any more, because configuration will be overridden by parameters in genesis block.